api_ui/api.yaml

```diff
@@ -1302,6 +1344,34 @@ components:
       items:
         $ref: '#/components/schemas/GetConnectionResponse'
 
+    AuthenticationConnection:
+      type: object
+      required:
+        - id
+        - userID
+        - issuerID
+        - createdAt
+        - modifiedAt
+      properties:
+        id:
+          $ref: '#/components/schemas/UUIDString'
```

Comment:
Don't you think it is better to return it as uuid instead of string as we do in other places. Like in Link
          type: string
          x-go-type: uuid.UUID
          x-go-type-import:
            name: uuid
            path: github.com/google/uuid
          example: 8edd8112-c415-11ed-b036-debe37e1cbd6